### PR TITLE
YAPR - Add routing

### PR DESCRIPTION
**Pull request by andresmoschini**
Yes, another PR.

I am introducing [React Router](https://reacttraining.com/react-router/), by the moment I am not doing more than redirect to _reports_ page when the root path is open.

Could you review?

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -47,14 +48,19 @@ class App extends Component {
...
-          <>
-            <Header userData={userData} />
-            <Reports />
-            <Footer />
-          </>
+          <Router>
+            <div>
+              <Header userData={userData} />
+              <Route path="/" exact component={redirectToReports} />
+              <Route path="/reports/" exact component={Reports} />
```
Comment (fcoronelmakingsense) [inline src/App.js]: in doppler the routes are in Title Clase like `Reports` . no need maintain consistency?''''''''''''

**Comment by fcoronelmakingsense** [comment]
Nice change 👍

File: src/App.js (hunk 1 above)
Comment (andresmoschini) [reply src/App.js]: Mmm, in general apps has lowercase URLs, in my opinion, we should take advantage of this opportunity to change it.

Comment (andresmoschini) [reply src/App.js]: I did not get answer on this, I think that in general Doppler's URLs are really ugly, so I do not see bad take this opportunity.

Merging...

**Pull request merged**